db/warehouse/migrate/20230406183420_client_mci_id.rb

```diff
@@ -0,0 +1,15 @@
+class ClientMciId < ActiveRecord::Migration[6.1]
+  def change
+    # create_table :external_ids do
+    #  t.string :external_id
+    #  t.references :owner, polymorphic: true # (Client, Project, etc.)
+    #  t.references :external_system # (MCI, MPER, etc.) (or could be a string..)
+    #  t.references :data_source
+    #  t.timestamps
```

Comment:
Based on our last conversation:
```suggestion
    #  t.string :value
    #  t.references :source, polymorphic: true
    #  t.references :remote_credential
    #  references :external_request_log
    #  t.timestamps
```

Theron's PR https://github.com/greenriver/hmis-warehouse/pull/2955 requires this table and the external driver to exist. I'm happy to merge this scaffolding in when it's ready @yakloinsteak and the remaining API bits can be finished up separately.

Reply:
I'm going to resolve this since I hadn't pushed my latest work yet. Sorry again about that.